meta-ostro/conf/local.conf.sample

```diff
@@ -291,6 +291,18 @@ SUPPORTED_RECIPES_CHECK = "fatal"
 #
 #require conf/distro/include/ostro-os-production.inc
 
+# systemd-bootchart is a useful tool to analyze and optimize a system
+# boot time. The tool is available in Ostro and needs to be activated
+# by a kernel command-line parameter which requires to build a new
+# image. You also need to make sure that you enable CONFIG_SCHEDSTATS=y
+# (that will also set CONFIG_SCHED_INFO=y) in your kernel, this option
+# is not enabled by default. For instructions on how to modify an Ostro
+# kernel, please check doc/howtos/modifying-ostro-kernel.rst.
```

Comment:
We could make it simpler to add CONFIG_SCHEDSTATS=y by providing a kernel config fragment for it. See meta-intel-iot-security/meta-security-framework/recipes-kernel/linux/linux-%.bbappend for an example where that is done for enabling audit.

Just a thought. We can also merge this as it is and enhance it later - @gvancuts, your call as the author of the PR.